web/cobrands/fixmystreet/assets.js

```diff
@@ -16,8 +16,58 @@ var fixmystreet = fixmystreet || {};
     };
 })();
 
+OpenLayers.Layer.VectorAsset = OpenLayers.Class(OpenLayers.Layer.Vector, {
+    initialize: function(name, options) {
+        OpenLayers.Layer.Vector.prototype.initialize.apply(this, arguments);
+        // Might only be able to fill in fields once they've been returned from the server
+        $(fixmystreet).on('report_new:category_change:extras_received', this.update_layer_visibility.bind(this));
```

Comment:
Do we think this is necessary? As I imagine the extras are only ever going to cut the number of bodies, will it matter if we're still loading both two-tier assets if present? I can think of a scenario where we would want to do so (a county asking for info on district related things, type thing).